tools/enrolment_preprocessor.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python3
+import csv
+import sys
+from argparse import ArgumentParser
+from dataclasses import dataclass
+
+from openpyxl import load_workbook
+from openpyxl.cell import Cell
+
+
+@dataclass
+class User:
+    title: str
+    last_name: str
+    first_name: str
+    email: str
+
+
+def fix_users(users: dict[str, User], title: Cell | None, last_name: Cell, first_name: Cell, email: Cell):
+    imported = User(last_name=last_name.value, first_name=first_name.value, email=email.value, title=title.value or "" if title else "")  # type: ignore[arg-type]  # if schema is correct, all values are strings
+    existing = users.setdefault(imported.email, imported)
+    if existing != imported:
+        print("There is a conflict in the user data.")
+        print(f"existing: {existing}.")
+        print(f"imported: {imported}.")
+        if input("Do you want to keep the existing user? (y/n)")[0].lower() == "n":
+            return
+
+        email.value = existing.email
+        last_name.value = existing.last_name
+        first_name.value = existing.first_name
+        if title:
+            title.value = existing.title
+
+
+if __name__ == "__main__":
+    args = sys.argv
+    if "python" in args[0]:
+        args = args[1:]
+    parser = ArgumentParser(description="Commandline tool to preprocess enrollment xlsx files.")
+    parser.add_argument(
+        "-ud", "--user-data", help="Path to a csv file containing an export of all existing users.", required=True
+    )
+    parser.add_argument(
+        "-ed", "--enrolment-data", help="Path to the enrolment data in xlsx format for import.", required=True
+    )
+    ns = parser.parse_args(args[1:])
```

Comment:
I've never seen anyone mess with `sys.argv` manually here. For me, locally, `python3` is stripped from `argv` if I run a script as `python3 test.py some_args`.